packages/roosterjs-content-model/lib/modelApi/common/mergeModel.ts

```diff
@@ -124,6 +124,10 @@ function mergeParagraph(
     if (segmentIndex >= 0) {
         newParagraph.segments.splice(segmentIndex, 0, ...newPara.segments);
     }
+
+    if (newPara.decorator) {
```

Comment:
What if we paste a paragraph with decorator into an existing paragarph without decorator?

Reply:
Checking what Word does:

If we use 'paste as is' and the target contains a decorator it is going to be overwritten by the source decorator. Eg: Current Content is H1, if we paste content with H2, the result decorator is going to be H2.

But if we do 'merge format' we should only keep the emphasis format. So in the above example the result decorator will continue being H1.